channels/urls.py

```diff
@@ -20,6 +22,16 @@
         name='post-list',
     ),
     url(
+        r'^api/v0/channels/(?P<channel_name>[A-Za-z0-9_]+)/moderators/$',
+        ModeratorListView.as_view(),
+        name='moderators-list',
```

Comment:
Can you change this to `moderator-list` and the below line to `moderator-detail`?